chains/manager.go

```diff
@@ -185,17 +195,19 @@ type manager struct {
 	// Key: Chain's ID
 	// Value: The chain
 	chains map[ids.ID]*router.Handler
+
+	// snowman++ related interface to allow validators retrival
+	validatorState validators.State
```

Comment:
Check how that was done before and if this breaks our way of injecting the static validator set.